mux.go

```diff
@@ -327,6 +336,19 @@ func mapFromPairs(pairs ...string) (map[string]string, error) {
 	return m, nil
 }
 
+func mapFromPairsToRegex(pairs ...string) (map[string]*regexp.Regexp, error) {
+	length, err := checkPairs(pairs...)
+	if err != nil {
+		return nil, err
+	}
+	m := make(map[string]*regexp.Regexp, length/2)
+	for i := 0; i < length; i += 2 {
+		regex, _ := regexp.Compile(pairs[i+1])
```

Comment:
I think this should check the error and panic. `regex` will be nil in the error case and the developer will get a panic when the mux tries to match, but it will be hard to trace down the root cause.

If this panics with a message, something like "invalid regular expression: %s" it will be obvious they made a mistake when setting up the routes in the first place.